src/datasets-string.c

```diff
@@ -98,6 +98,12 @@ uint32_t StringHash(void *s)
     return hash;
 }
 
+uint32_t StringGetLength(void *s)
```

Comment:
Could this be `size_t` to be consistent with the size used in `StringAsBase64`?

Reply:
no size_t please, we use explicit sizes as much as possible